src/database/seeders/20201223133014-User.js

```diff
@@ -8,13 +8,33 @@ module.exports = {
       password: '1234567$#8',
       gender: 'Male',
       origin: '',
+      profession: '',
```

Comment:
Please do not consider this field of profession, it was replaced by managerId.
Line 11 and 27.

Reply:
I didn't use the user's seeder sir